pyclesperanto/_categories.py

```diff
@@ -0,0 +1,17 @@
+def categories():
+
+    from inspect import getmembers, isfunction
```

Comment:
```suggestion
from inspect import getmembers, isfunction
 
def categories():   
```

I would put the import outside of the function if possible